react/src/views/Transfers/CreateTransferShipment/components/CreateTransferShipment.tsx

```diff
@@ -0,0 +1,254 @@
+          data: {
+            agreementId: partnerBasesDataForSelectedOrganisation.agreementId,
+            specialNote: partnerBasesDataForSelectedOrganisation.specialNote,
+          },
+        });
+      }
+    }
+  }, [partnerOrganisation, resetField, setValue, partnerOrganisationsAgreementsData]);
+
+  return (
+    <Box w={["100%", "100%", "60%", "40%"]}>
+      <Heading fontWeight="bold" mb={8} as="h1">
+        Start New Shipment
+      </Heading>
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <List spacing={2}>
+          <ListItem>
+            <HStack mb={4}>
+              <SendingIcon />
+              <Text fontWeight="bold" fontSize="md">
+                {" "}
+                SENDING
+              </Text>
+            </HStack>
+          </ListItem>
+          <ListItem>
+            <Center>
+              <Text fontWeight="medium" fontSize="md">
+                {currentOrganisationLabel}
+              </Text>
+            </Center>
+          </ListItem>
+        </List>
+        <Spacer mb={8} />
+        <Divider size="2" borderColor="black" />
+        <Spacer mb={8} />
+        <List spacing={2}>
+          <ListItem>
+            <HStack mb={4}>
+              <ReceivingIcon />
+              <Text fontWeight="bold" fontSize="md">
+                {" "}
+                RECEIVING
+              </Text>
+            </HStack>
+          </ListItem>
+          <ListItem>
+            <SelectField
+              fieldId="partnerOrganisation"
+              fieldLabel="Organisation"
+              placeholder="Please select an organisation"
+              options={partnerOrganisationsForDropdownGroups}
+              errors={errors}
+              control={control}
+            />
+          </ListItem>
+          <ListItem>
+            <SelectField
+              fieldId="partnerOrganisationSelectedBase"
+              fieldLabel="Base"
+              placeholder="Please select a base"
+              errors={errors}
+              control={control}
+              options={basesOptionsForPartnerOrg}
+            />
+          </ListItem>
+          {/* <ListItem>
+            <Text>Agreement #{} Comment</Text>
+          </ListItem> */}
+        </List>
+
+        <Stack spacing={4}>
+          <ButtonGroup gap="4">
+            <Button
+              mt={10}
+              isLoading={isSubmitting}
+              size="md"
+              type="button"
+              borderRadius="0"
+              w="full"
+              variant="outline"
+              onClick={() => navigate(`/bases/${baseId}/transfers/shipments`)}
+            >
+              Cancel
+            </Button>
+
+            <Button
+              mt={10}
+              isLoading={isSubmitting}
+              type="submit"
+              borderRadius="0"
+              w="full"
+              variant="solid"
+              backgroundColor="blue.500"
+              color="white"
```

Comment:
why white color?

Reply:
@HaGuesto The UI should display the submit button text color in white, but I think we can have a general submit button within the theme so we don't need to apply style to it